lib/inspec/resources/processes.rb

```diff
@@ -60,6 +60,15 @@ def list
       @list
     end
 
+    # Matcher to check if the process is running
+    def running?
+      # Check if Regex needs to be tightened.
+      # States value can be as:
+      # for Unix: R, R< or R+
+      # for Windows "True" or "False"
+      states.any? and !!(states[0] =~ /True/ || states[0] =~ /^R+/)
```

Comment:
That's wrong. You probably don't want literally R for running - Sleep is probably OK too. Interpret this as "is the process alive, not dead"

Reply:
Changed logic to consider sleep state (both uninterruptible and interruptible) to be running.